cxflow/hooks/stop_after.py

```diff
@@ -31,7 +30,7 @@ class StopAfter(AbstractHook):
               iterations: 1000
     """
 
-    def __init__(self, epochs: Optional[int]=None, iterations: Optional[int]=None, minutes: Optional[float]=None,
+    def __init__(self, epochs: Optional[int]=None, iterations: Optional[int]=None, minutes: Optional[float]=None, train_stream: str="train",
```

Comment:
- Line too long.
- This project uses single quotes instead of double quotes. Please use those for the sake of consistency.